.gitignore

```diff
@@ -25,3 +25,4 @@ env/
 /resources/plugins/pretense/pretense_output.lua
 
 *.psd
+.pre-commit-config.yaml
```

Comment:
No need to ignore this file, I use a different approach, hence it probably stuck with an old version 👍